webapp/WEB-INF/templates/admin/user/attribute/checkbox/html_code_form_attribute_checkbox.html

```diff
@@ -19,11 +19,8 @@
 			<#if attribute.fieldInLine>
 				<#assign orientation = 'horizontal'>
 			</#if>
-			<#if attributeField.defaultValue >
-				<#assign checked = true>
-			</#if>
 		
-			<@checkBox name='attribute_${attribute.idAttribute}' value='${attributeField.idField}' checked=checked labelKey='${attributeField.title}' orientation=orientation />

```

Comment:
Doesn't this remove the functionality of the default value ? That's what is currently happening with comboxes, and this commit switches checkboxes and radios to the same method.